components/src/lib/ContentModal/ContentModal.stories.tsx

```
@@ -0,0 +1,30 @@
+const meta = {
+  title: "Component/ContentModal",
+  component: ContentModal,
+  tags: ["autodocs"],
+  args: {
+    title: "ContentModal Title",
+    content: <p>ContentModal Content</p>,
+    hasCloseButton: true,
+    containerAs: "div",
+    size: "medium",
+    buttonElements: [
+      <Button variant="secondary" type="button">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)